pkgs/by-name/kr/kryptor/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+}:
+
+buildDotnetModule rec {
+  pname = "kryptor";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "samuel-lucas6";
+    repo = "Kryptor";
+    rev = "v${version}";
```

Comment:
```suggestion
    rev = "refs/tags/${version}";
```

Reply:
I don't think this change is necessary and also don't think this works, as the version has a v in the beggining.